src/main/pages/che-7/overview/proc_registering-to-hosted-che.adoc

```diff
@@ -15,7 +15,7 @@ This section describes how to register to Hosted{nbsp}Che.
 
 . Navigate to link:https://che.openshift.io/[Hosted{nbsp}Che].
 
-. Log in with the existing OpenShift{nbsp}Online, Red{nbsp}Hat Developer{nbsp}Program, or Red{nbsp}Hat Customer{nbsp}Portal account.
+. Log in with the existing OpenShift{nbsp}Online, Red{nbsp}Hat Developer{nbsp}Program, or Red{nbsp}Hat Customer{nbsp}Portal account, *or* register for a new Red{nbsp}Hat account.
```

Comment:
@ibuziuk the 'Prerequisites' state:

`* Existing OpenShift{nbsp}Online, Red{nbsp}Hat Developer{nbsp}Program, or Red{nbsp}Hat Customer{nbsp}Portal account.`

This addition seems like a repetition.

Reply:
@rkratky  'Prerequisites' is utterly misleading IMO. One should not have `existing Existing OpenShift{nbsp}Online, Red{nbsp}Hat Developer{nbsp}Program, or Red{nbsp}Hat Customer{nbsp}Portal account` before navigating to che.openshift.io - an account can be created right away on the https://developers.redhat.com/ where user will be redirected from he 
che.openshift.io

Reply:
Oh that's good news, then can you please also remove the prerequisite in that PR @ibuziuk. I am sorry I introduced this misleading information.

Reply:
@themr0c @rkratky PR updated, ambiguous 'Prerequisites' have been removed